modules/nf-core/limma/differential/templates/limma_de.R

```diff
@@ -287,11 +295,38 @@ colnames(design) <- sub(
     paste0(contrast_variable, '.'), colnames(design)
 )
 
+# Perform voom normalisation for RNA-seq data
+if (!is.null(opt\$use_voom) && opt\$use_voom) {
+    # Create a DGEList object for RNA-seq data
+    dge <- DGEList(counts = intensities.table)
+
+    # Normalize counts using TMM
+    dge <- calcNormFactors(dge, method = "TMM")
+
+    # Run voom to transform the data
+    voom_result <- voom(dge, design)
+    data_for_fit <- voom_result
+
+    # Write the normalized counts matrix to a TSV file
+    normalized_counts <- voom_result\$E
```

Comment:
If we're adding export of normalised values (which I think is a good thing) we should do the same for the microarry (e.g. with `normalizeBetweenArrays`), and do the writing outside this conditional.

Reply:
Sure will try to add. - If I will have some problems I will let you know

Reply:
If I understand correctly, the assumption in limma for microarray analysis is that the input intensity table to this module has already been normalized. Therefore, we just need to write the normalized intensity table which was the input, and there's no need to apply `normalizeBetweenArrays`.

Reply:
Oh yes, you're right actually- my bad, apologies @KamilMaliszArdigen, you can disregard this comment. 

See https://github.com/nf-core/differentialabundance/blob/3dd360fed0dca1780db1bdf5dce85e5258fa2253/workflows/differentialabundance.nf#L307.

Reply:
That actually makes more sense :)

Reply:
To be doubly clear, I take back the comment- no normalised table required for the array mode :-). That output will be optional.